ruby_event_store_rom_sql/ruby_event_store_rom_sql.gemspec

```diff
@@ -0,0 +1,38 @@
+# coding: utf-8
+lib = File.expand_path('../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'ruby_event_store/rom/version'
+
+Gem::Specification.new do |spec|
+  spec.name          = 'ruby_event_store_rom_sql'
+  spec.version       = RubyEventStore::ROM::VERSION
+  spec.licenses      = ['MIT']
+  spec.authors       = ['Joel Van Horn']
+  spec.email         = ['joel@joelvanhorn.com']
```

Comment:
I haven't thought much about it yet however we'll likely need `dev@arkency.com` user to be gem owner as well to handle releases of all gems at once smoothly.